ccm_web/client/src/utils/GradebookProcessor.tsx

```diff
@@ -0,0 +1,141 @@
+import { CSVRecord } from '../utils/FileParserWrapper'
+import { InvalidationType } from '../models/models'
+
+export interface GradebookUploadRecord extends CSVRecord {
+  'SIS Login ID': string
+}
+
+export const REQUIRED_LOGIN_ID_HEADER = 'SIS Login ID'
```

Comment:
I have seen the text 'SIS Login ID` at-least 3 places here. I would suggest make it a constant variable and use it. Same with 'Point Possible` string

Reply:
Alright, this was already kind of a global, but I found a couple more places where it could be used. I also created `POINTS_POS_TEXT`, see https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/223/commits/ae5c9174a772f09eee36a9e685919795342bf7bd